muxers/mplex/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.43.1
```

Comment:
This should either be increased to 0.44.0 or there should be `MplexConfig` type alias with deprecation message for backward compatibility. Right now the changes break existing API.

Reply:
I would advise to use an alias with a deprecation message first